treyholt/Suzie
# Bringing Jeans, T-Shirts and Shirts to Zara

This PR shows the tweaks made to upload a bunch of reformation jeans to our backend. 

It also holds the client side code to now be able to parse jeans, t-shirts or default shirts. 

To Test: 

Run the extension like normal, but please test ruthlessly to confirm it all still works well. 
Make sure when you go on a jeans it recommends jeans and not tops and vice versa.
